### DISCUSSION TITLE
actual User Guide

### DISCUSSION THREAD

**Discussion by PatMoonB** [aclap-dev/video-downloadhelper#870] [Help and Q&A]
### Your question

Where can I find an actual User Guide?
I'm iterested in the the workflow, when downloading a file:
Sometimes I get a scrowling progress bar, sometimes a fixed and slowly progressen one.
What's the disserence?
With the first there is no info, what time it will take to finish, only a byterate and a time the download strted.
With the second I get an info with percentage downloaded and an estamate.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/167686372/270eb055-9b8a-48a7-8d25-339e03d8af33)
Thx PatMoonB

### Addon & CoApp version and details

Version 9.0.2.3 Date 29. May 2024

version: 9.0.2.3
target: google
channel: stable
lang: de
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"da9c3943d7d648d3"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\PT\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 44
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

It is my understanding that it all depends on the way the web site serves its information, not just to VDH but to any client browser.  Sometimes there's enough information to report percentage complete & duration of the object, sometimes not.  For example, with a livestream, there's no way to know when it will end so the best you can do is say how long you've been reading it.  But other factors can affect what VDH reports.  VDH can report only what it knows, & sometimes it doesn't know much.

**Reply by PatMoonB** [reply]
That could be so, or not.
I thhink VDH knows the byte count of the file its koading, in this case a *.webm.
I will look after a *.mp4 file next time and let you know if that is the second situation, with a stable scrollbar and a percentage ;(

But do you know the actual 9.0 DWH user guide?
Where can I downkiad this? I only have a Version 7 or 8 one.

**Reply by PatMoonB** [reply]
Your write, the second scrollbar appears when you koad a MP4 File
![image](https://github.com/aclap-dev/video-downloadhelper/assets/167686372/f5be48c7-5b37-4529-9357-7218c9a9aabd)

**Reply by GrampaWildWilly** [reply]
You know it's dangerous to jump to conclusions based on only a single data point.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/df84f843-e3c7-42bb-981e-0e8f4b2bf7ff)

I'm going to download this video as MKV.

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/80c9cd9e-18ba-4121-aafc-34cc8e254a2c)

Here's the progress status menu for this download.  Looks like all the usual bits of information are there.

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/959a21cd-fe09-439a-9204-12eaa2614486)
![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e0d03312-486f-4a5e-91d3-2cba9475c8c8)

This is what downloaded:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/12204bcb-82f9-4f6d-9541-9e772d56b3ee)

Here it is playing in VLC.

![#06](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8959f31f-b4b3-43d2-9419-695f2fce9df5)

You can see it looks just fine.  You'll have to take my word for it that it sounds just fine, too.

When it comes to a user guide, your best bet is to click the Wiki link at the top of any page of this forum.  I suspect that whatever you downloaded is not actually for VDH 7 or 8 but more likely VDH 5.  That was the latest thing they had on the VDH web site last time I looked.  In other words, it contains only stuff that is vaguely applicable today, but mostly not.  They have redone the web site more recently, it seems.  I don't see anything resembling what I am remembering & what you must have downloaded.  I say you can safely trash whatever it is you have because it is woefully out of date.

**Reply by GrampaWildWilly** [reply]
By the way, your German web site is mercilessly geoblocking me.  I can't test out anything on that site.  And since you're using Chrome, you can't test out downloads on YouTube, which is my go-to site for testing because it's got videos & I'm not geoblocked there.

Oh yeah.  I should have mentioned that I am running Windows 11 Pro 64-bit version 23H2 build 22631.3737, Firefox 127.0 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

**Reply by PatMoonB** [reply]
Hi GrampaWW, thx for your effort in reproducing my site problem.
You are right that it is wrong to judge the scrollbar problem by one file extension.
Now I get the straight scrollbar with WEBM, too :)
It seems the topic is my slow / unsteady internet I got at home. When I'm sitting almost under the WLAN port, DWH is finding all data of the downloading file, if not it doesn't find it;)

Sorry for using chrome (I_m too old to change until I'll have to swap to Win11 for God's sake) and the geo-blocking germans are using (damn data policy in EU)

So I close this post and write to the support address for my request to get out an actual user guide for download directly by mail.
Thx again and till the next time
PatMoonB
